README.org

```diff
@@ -103,6 +108,10 @@ have reproducable builds, you may generate one by running:
 #+END_SRC
 
 * Changelog
+* 1.2.0
+TBD
```

Comment:
We can set this when we're ready to cut a release! And if we want to use this PR for the release we can just add it via commit before merging.